querysync/javatests/com/google/idea/blaze/qsync/testdata/kotlinlib/BUILD

```diff
@@ -1,4 +1,4 @@
-load("//tools/build_defs/kotlin:rules.bzl", "kt_jvm_library")
+load("@rules_kotlin//kotlin:jvm.bzl", "kt_jvm_library")
```

Comment:
Seems like `tools/build_defs/kotlin:rules.bzl` is no longer used. Could we delete it?

Reply:
it's already gone